common/src/main/java/org/cloudfoundry/identity/uaa/login/saml/IdentityProviderDefinition.java

```diff
@@ -40,6 +40,7 @@
     private String socketFactoryClassName;
     private String linkText;
     private String iconUrl;
+    private boolean addNew = true;
```

Comment:
A more descriptive name would help here. a name like addShadowAccountUponSamlLogin ensures there is no confusion.